eng/CredScanSuppression.json

```diff
@@ -25,6 +25,14 @@
       "hash": "5ETIzLYe9aa9Xx5512Uy6gJiCZ/CL2QzS3nDfN51nz8=",
       "_justification": "Secret used by test code"
     },
+    {
+      "placeholder": "Resource",
```

Comment:
Why is this getting flagged?

Reply:
Can we use an existing placeholder instead?

Reply:
This is a false positive in the generated api review file for this line: `export type Secret = Resource & {` (sdk/keyvault/arm-keyvault/review/arm-keyvault.api.md) I don't think we can or want to modify this file.

Reply:
https://github.com/Azure/azure-sdk-for-js/blob/de5a026e23211d1641d319dbd5d7a12710215953/sdk/keyvault/arm-keyvault/review/arm-keyvault.api.md#L1150

Reply:
OK lets combine these into the one list then.